src/components/Table/GridTable.tsx

```diff
@@ -854,6 +854,7 @@ export type GridCellContent = {
   /** Whether to indent the cell. */
   indent?: 1 | 2;
   colspan?: number;
+  css?: Properties;
```

Comment:
Ah yeah; I think historically I would have pushed to keep this as just conditional logic in the `GridRowStyle[kind].cellCss(row)` lambda...

If anything to kind of purposefully nudge the user to "just use the provided GridStyle and don't do customize UI work" ... this almost makes it too easy. :-D 

Just curious, but is this solely for text sizing? Wdyt about adding like a `typeScale` instead of a completely open-ended `css` prop? Just thinking by staying more restricted on what we allow here (and have the user declare intent vs. raw css), it will give us more flexibility to change the internals...

That said, if you already have "more than just text" that you need this for, and yeah this is admittedly already possible with `GridRowStyle.cellCss`, just a different/easier way to skin the cat, I'm good with it too.

Reply:
I don't believe this is actually possible with the `GridRowStyle.cellCss`, as that targets _every_ cell in the row, where I only want to target a specific column's cell for this row kind.

And yes, I was just looking to do typeScale for the moment, so I could limit to just that for now. Though, allowing this could also remove the need for the `alignment` and `indent` props, as a user could specify that sort of thing via CSS and have more flexibility on it. Something to think about.

Reply:
> as that targets every cell in the row

Oh shoot, you're right, I was getting columns & kinds mixed up. Gotcha!